### DISCUSSION TITLE
database error when updating event

### DISCUSSION THREAD

**Discussion by StickyPixels** [liedekef/events-made-easy#284] [General]
Hi Franky,
Whenever I update an existing event I get a php error warning:
`WordPress databasefout: [Table 'database_name.t7v3_eme_task_signups' doesn't exist]
DELETE FROM t7v3_eme_task_signups WHERE event_id=188`
When saving a recurrent event I get as many warnings as there are recurrences.
The update is saved correctly and there are no other issues, so it's just an annoying message. I have tasks disabled, so I assume it has something to do with our custom database prefix but I'm really no expert. Hopefully you can make sense of it?

**Comment by liedekef** [comment]
If the table doesn't exist, that will be a problem. See this : https://github.com/liedekef/events-made-easy/issues/274 and the fix the person found: https://github.com/liedekef/events-made-easy/issues/274#issuecomment-1686200458

**Comment by StickyPixels** [comment]
Worked perfectly, I updated EME to the latest version and did the trick with database version. Now the message doesn't appear anymore.
Many thanks!
